superset-frontend/src/views/CRUD/welcome/SavedQueries.tsx

```diff
@@ -0,0 +1,260 @@
+import Icon from 'src/components/Icon';
+import SubMenu from 'src/components/Menu/SubMenu';
+import EmptyState from './EmptyState';
+
+import { IconContainer, CardContainer, createErrorHandler } from '../utils';
+
+const PAGE_SIZE = 3;
+
+interface Query {
+  id?: number;
+  sql_tables?: Array<any>;
+  database?: {
+    database_name: string;
+  };
+  rows?: string;
+  description?: string;
+  end_time?: string;
+  label?: string;
+}
+
+interface SavedQueriesProps {
+  user: {
+    userId: string | number;
+  };
+  queryFilter: string;
+  addDangerToast: (arg0: string) => void;
+  addSuccessToast: (arg0: string) => void;
+}
+
+const QueryData = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
+  .title {
+    font-weight: ${({ theme }) => theme.typography.weights.normal};
+    color: ${({ theme }) => theme.colors.grayscale.light2};
+  }
+  .holder {
+    margin: ${({ theme }) => theme.gridUnit * 2}px;
+  }
+`;
+const SavedQueries = ({
+  user,
+  addDangerToast,
+  addSuccessToast,
+}: SavedQueriesProps) => {
+  const {
+    state: { loading, resourceCollection: queries },
+    hasPerm,
+    fetchData,
+    refreshData,
+  } = useListViewResource<Query>('saved_query', t('query'), addDangerToast);
+  const [queryFilter, setQueryFilter] = useState('Favorite');
+  const [queryDeleteModal, setQueryDeleteModal] = useState(false);
+  const [currentlyEdited, setCurrentlyEdited] = useState<Query>({});
+
+  const canEdit = hasPerm('can_edit');
+  const canDelete = hasPerm('can_delete');
+
+  const handleQueryDelete = ({ id, label }: Query) => {
+    SupersetClient.delete({
+      endpoint: `/api/v1/saved_query/${id}`,
+    }).then(
+      () => {
+        refreshData();
+        setQueryDeleteModal(false);
+        addSuccessToast(t('Deleted: %s', label));
+      },
+      createErrorHandler(errMsg =>
+        addDangerToast(t('There was an issue deleting %s: %s', label, errMsg)),
+      ),
+    );
+  };
+
+  const getFilters = () => {
+    const filters = [];
+    if (queryFilter === 'Mine') {
+      filters.push({
+        id: 'created_by',
+        operator: 'rel_o_m',
+        value: `${user?.userId}`,
+      });
+    } else {
+      filters.push({
+        id: 'id',
+        operator: 'saved_query_is_fav',
+        value: true,
+      });
+    }
+    return filters;
+  };
+
+  useEffect(() => {
+    fetchData({
+      pageIndex: 0,
+      pageSize: PAGE_SIZE,
+      sortBy: [
+        {
+          id: 'changed_on_delta_humanized',
+          desc: true,
+        },
+      ],
+      filters: getFilters(),
+    });
+  }, [queryFilter]);
+
+  const renderMenu = (query: Query) => (
+    <Menu>
+      {canEdit && (
+        <Menu.Item
+          onClick={() => {
+            window.location.href = `/superset/sqllab?savedQueryId=${query.id}`;
+          }}
+        >
+          {t('Edit')}
+        </Menu.Item>
```

Comment:
```tsx
        <Menu.Item>
         <a href={`/superset/sqllab?savedQueryId=${query.id}`} target="_blank">{t('Edit')}</a>
        </Menu.Item>
```

Why not just a regular anchor element (just like [AntD's official example](https://ant.design/components/dropdown/))? It's more semantically correct and better for a11y. Users can also "cmd + click" to open things in new tabs if they want to.

Reply:
Regular anchor element might have an underscore. So #11491 is needed.